lib/sanitize.js

```diff
@@ -73,5 +73,41 @@ sanitizer.config = {
     // Allow YouTube iframes
     if (frame.tag !== 'iframe') return false
     return !String(frame.attribs.src).match(/^(https?:)?\/\/(www\.)?youtube\.com/)
+  },
+  transformTags: {
+    'td': sanitizeCellStyle,
+    'th': sanitizeCellStyle
+  }
+}
+
+// Allow table cell alignment
+function sanitizeCellStyle (tagName, attribs) {
+  // if we don't add the 'style' to the allowedAttributes above, it will be
+  // stripped out by the time we get here, so we have to filter out
+  // everything but `text-align` in case something else tries to sneak in
+  function cell (alignment) {
+    var attributes = attribs
+    if (alignment) {
+      attributes.style = 'text-align:' + alignment
+    } else {
+      delete attributes.style
+    }
+    return {
+      tagName: tagName,
+      attribs: attributes
+    }
+  }
+
+  // note: we can do simple .indexOf checks here because markdown-it always
+  // generates `text-align` styles without whitespace
+  var style = attribs.style || ''
+  if (style.indexOf('text-align:left') >= 0) {
+    return cell('left')
+  } else if (style.indexOf('text-align:right') >= 0) {
+    return cell('right')
+  } else if (style.indexOf('text-align:center') >= 0) {
+    return cell('center')
+  } else {
+    return cell()
```

Comment:
@ashleygwilliams as you can see here and in the unit tests, markdown-it always generates the `text-align` stuff with no whitespace between the colon and the property value, so I'm just checking literally for that. A potential downside of doing it this way is that inline or hand-written HTML tables with cell alignments that look like, e.g., `text-align: center` (i.e, with a space in the middle) will get stripped out even though they're basically benign.

I have a unit test that specifically calls attention to this, to make it clear that the behavior is intentional. Should this maybe be a regular expression match with optional whitespace in the middle?